packages/app/src/App.tsx

```diff
@@ -293,5 +294,6 @@ export default app.createRoot(
     <AppRouter>
       <Root>{routes}</Root>
     </AppRouter>
+    <AppRatingPopUp title="Rate Backstage" chance={100} />
```

Comment:
Super nit, but could you move this up above the `AppRouter` with the rest of the app-wide dialogs?